Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/SegmentEditorThresholdEffect.py

```diff
@@ -72,11 +75,11 @@ def icon(self):
         return qt.QIcon()
 
     def helpText(self):
-        return """<html>Fill segment based on source volume intensity range<br>. Options:<p>
+        return _("""<html>Fill segment based on source volume intensity range<br>. Options:<p>
 <ul style="margin: 0">
 <li><b>Use for masking:</b> set the selected intensity range as <dfn>Editable intensity range</dfn> and switch to Paint effect.</li>
 <li><b>Apply:</b> set the previewed segmentation in the selected segment. Previous contents of the segment is overwritten.</li>
-</ul><p></html>"""
+</ul><p></html>""")
```

Comment:
```suggestion
</ul></p></html>""")
```

Reply:
I would vote for keeping the text simpler and omit the optional paragraph closing tag (unless there is a specific reason to add it).

Reply:
Doing this would allow to avoid having the `<html>` tag in the string to translate:
```
return "<html>" + _("""Text to translate.""")
```

Reply:
@lassoan Would you like to do these tweaks in this pull request or later ? Based on your answer, I will approve (or not)